readme.md

```diff
@@ -84,6 +83,15 @@ Supported URLs / URIs:
 *   `./image.jpg`
 *   `../image.jpg`
 
+##### `options`
+
+Configuration.
+
+###### `options.imageExtensions`
+
+List of file extensions recognized as images (`Array.<string>?`, default
+`['svg', 'png', 'jpg', 'jpeg', 'gif', 'webp', 'avif']`).
+
```

Comment:
When `defaultImageExtensions` is documented, this list can be replaced with a link.

Might be good to state that these extnames must not use `.`

Reply:
noted